native/src/modules/categories/components/Categories.js

```diff
@@ -60,34 +55,19 @@ class Categories extends React.Component<PropsType> {
   }
 
   navigateToFeedback = (isPositiveFeedback: boolean) => {
-    const { navigation, t, stateView, cities, cityCode, language } = this.props
-    if (!cityCode || !language) {
-      throw Error('language or cityCode not available')
-    }
-
-    const createFeedbackVariant = (
-      label: string, feedbackType: FeedbackType, feedbackCategory: FeedbackCategoryType, pagePath?: string
-    ) => new FeedbackVariant(label, language, cityCode, feedbackType, feedbackCategory, pagePath)
-    const cityTitle = CityModel.findCityName(cities, cityCode)
+    const { navigation, stateView, cityCode, language } = this.props
     const category = stateView.root()
 
-    const feedbackItems = [
-      createFeedbackVariant(t('feedback:contentOfCity', { city: cityTitle }),
-        CATEGORIES_FEEDBACK_TYPE, CONTENT_FEEDBACK_CATEGORY),
-      createFeedbackVariant(t('feedback:technicalTopics'), CATEGORIES_FEEDBACK_TYPE, TECHNICAL_FEEDBACK_CATEGORY)
-    ]
-
-    if (!category.isRoot()) {
-      feedbackItems.unshift(
-        createFeedbackVariant(t('feedback:contentOfPage', { page: category.title }),
-          PAGE_FEEDBACK_TYPE, CONTENT_FEEDBACK_CATEGORY, category.path)
-      )
+    const feedbackInformation: FeedbackInformationType = {
+      type: 'Category',
+      language,
+      cityCode,
+      title: !category.isRoot() ? category.title : undefined,
+      path: !category.isRoot() ? category.path : undefined,
+      isPositiveFeedback
     }
 
-    navigation.navigate('FeedbackModal', {
-      isPositiveFeedback,
-      feedbackItems
-    })
+    navigation.navigate('FeedbackModal', { ...feedbackInformation })
```

Comment:
Why don't you use `createNavigateToFeedbackModal` here?

Reply:
good question. Earlier I was also fetching the cities in createNavigateToFeedbackModal. As I removed it, I will do as you say.